### DISCUSSION TITLE
How create TableSpace in RAM drive(mount tmpfs or rmpfs) via HammerDB tcl

### DISCUSSION THREAD

**Discussion by arjunshetty955** [TPC-Council/HammerDB#325] [General]
How to create TableSpace in RAM drive(mount tmpfs or rmpfs) via HammerDB tcl
And I tired manually with postgres commands and it not helpemout 

Example:
postgres@centos8pg#sudo mkdir /my_ram_disk_2/
postgres@centos8pg#sudo mount -t tmpfs -o size=2G tmpfs /my_ram_disk_2/
postgres@centos8pg# df -h | grep disk_2
tmpfs                2.0G     0  2.0G   0% /my_ram_disk_2 
postgres=# \! sudo chown postgres:postgres /my_ram_disk_2/
postgres=# \! mkdir /my_ram_disk_2/tbs2
postgres=# create tablespace tpcc location '/my_ram_disk_2/tbs2'

the intent to check to RAM Drive for tests or benchmark on  large RAM disk to avoid wasting of SDDs and create test dbs on RAM drive.
Pg_wal can be create in RAM Drive ?

Reference link: Benchmark comparison btw RAM drive and SSD
https://tech.marksblogg.com/test-django-on-ram-drive.html

**Comment by sm-shaw** [comment]
hammerDB supports using PostgreSQL tablespaces, the  default tablespace is pg_default. If you want the WAL in a specific location, use the --waldir=directory option of initdb.    How and where this underlying storage is configured is out of scope for HammerDB support.  This question would be best directed to a PostgreSQL forum.
